index.bs

```diff
@@ -10562,14 +10562,18 @@ Operations that happen <dfn>atomically</dfn> on a
 given thread can only be executed when no other [=Atomically|atomic=]
 operation is running on another thread.
 
-The algorithm for rendering a block of audio from an
-{{AudioContext}} <em>G</em> with a <a href="#control-message-queue">control message queue</a> <em>Q</em> is as
-follows.
-
-If the algorithm returns true then it MUST be executed again in the
-future, to render the next block of audio. Else, the <a>rendering
-thread</a> yields and the processing stops. The <a>control thread</a>
-can restart executing this algoritm if needed.
+The algorithm for rendering a block of audio from an {{AudioContext}} <em>G</em>
+with a <a href="#control-message-queue">control message queue</a> <em>Q</em> is
+comprised of 3 steps and the order of these steps cannot be altered or
+interrupted:
+	1. Process the control message queue.
+	2. Process a render quantum.
+	3. Perform a microtask check point.
+
+If the second step returns true then it MUST be executed again in the future, to
```

Comment:
This is kind of inconsistent with what line 10567 says about not being altered or interrupted.

Reply:
I revised the structure to address this.